lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,98 @@ def _get_default_parameters(kymo, channel):
 
     return {
         "pixel_threshold": KymotrackerParameter(
-            "Threshold",
-            "Set the pixel threshold.",
+            "Minimum intensity",
+            "Set the intensity threshold.",
```

Comment:
I think here it's still ok to keep the reference to "pixel", it does make it clear what is actually being checked
```suggestion
            "Minimum intensity",
            "Set the pixel intensity threshold.",
```

Reply:
👍